README.md

```diff
@@ -2,59 +2,127 @@
 
 ## Overview
 
-This is a tool for transforming and processing VCF files in a scalable manner
-based on [Apache Beam](https://beam.apache.org/). The Python implementation
-supports running pipelines using
+This is a tool for transforming and processing
+[VCF](https://samtools.github.io/hts-specs/VCFv4.3.pdf) files in a scalable
+manner based on [Apache Beam](https://beam.apache.org/) using 
 [Dataflow](https://cloud.google.com/dataflow/) on Google Cloud Platform.
 
-### Pre-requisites
+It can be used to directly load VCF files to
+[BigQuery](https://cloud.google.com/bigquery/) in a scalable manner supporting
```

Comment:
Remove "in a scalable manner" as its already mentioned above.

Reply:
Done. Good idea!